patroni/ctl.py

```diff
@@ -3,39 +3,44 @@
 '''
 
 import click
-import os
-import yaml
+import datetime
+import dateutil
 import json
-import time
+import logging
+import os
 import psycopg2
 import random
 import requests
-import datetime
-from prettytable import PrettyTable
-from six.moves.urllib_parse import urlparse
-import logging
-import dateutil
+import time
 import tzlocal
+import yaml
```

Comment:
Since you reshuffle the imports, I'd suggest to follow PEP8 and separate the built-in modules, 3rd party (dateutil, psycopg2, request, tzlocal, yaml) and the app-specific ones, as PEP8 suggests:

Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.